### PR TITLE
emailTests

### PR DESCRIPTION
**Pull request by ItzikShiva**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -24,7 +24,7 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5">
```
Comment (hodkash) [inline .classpath:27]: why?

**Review by hodkash — APPROVED** [review]

File: src/main/java/absproj/email/api/EmailService.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import com.google.gson.Gson;
+
+public class EmailService {
+
+	public CloseableHttpResponse getEmailValidation(String baseURI, String api_key, String email)
```
Comment (hodkash) [inline src/main/java/absproj/email/api/EmailService.java:17]: Add public CloseableHttpResponse getEmailValidation(String email)
Add public CloseableHttpResponse getEmailValidation(String apiKey, String email)

File: src/test/java/absproj/api/EmailTests.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+		logger.info("got invalidEmailTest - success");
+
+		EmailResponse emailResponse = httpResponseToObj(httpResponse, EmailResponse.class);
+
+		assertEquals(emailResponse.getDeliverability(), "UNDELIVERABLE");
+		logger.info("got \"UNDELIVERABLE\" success result");
+	}
+
+	@Test
+	public static void emaptyEmailTest() throws ClientProtocolException, IOException {
```
Comment (hodkash) [inline src/test/java/absproj/api/EmailTests.java:79]: typo

File: src/main/java/absproj/email/api/EmailService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ItzikShiva) [reply src/main/java/absproj/email/api/EmailService.java:17]: resolve

File: src/test/java/absproj/api/EmailTests.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+				emailParams.valid_api_key, emailParams.valid_email);
+		assertEquals(httpResponse.getStatusLine().getStatusCode(), 200);
+		logger.info("got valid email response from server");
+
+		EmailResponse emailResponse = httpResponseToObj(httpResponse, EmailResponse.class);
+		assertEquals(emailResponse.getDeliverability(), "DELIVERABLE");
+		logger.info("got valid DELIVERABLE email");
+		assertTrue(emailResponse.getIs_valid_format().getValue());
+		logger.info("got valid email format");
+
```
[the remark this answers is not in the document]
Comment (ItzikShiva) [reply src/test/java/absproj/api/EmailTests.java:47]: done, to show HOD that i found error in the my POJO because of this!

**Pull request merged**